frontend/src/views/auth/Signup.vue

```diff
@@ -79,17 +98,36 @@
                 }}</span
               >
             </label>
-            <div class="mt-1 rounded-md shadow-sm">
+            <div
+              class="relative flex flex-row items-center mt-1 rounded-md shadow-sm"
+            >
               <input
                 id="password-confirm"
                 v-model="state.passwordConfirm"
-                type="password"
+                :type="state.showPassword.confirm ? 'text' : 'password'"
                 autocomplete="off"
                 :placeholder="$t('auth.sign-up.confirm-password-placeholder')"
                 required
                 class="appearance-none block w-full px-3 py-2 border border-control-border rounded-md placeholder-control-placeholder focus:outline-none focus:shadow-outline-blue focus:border-control-border sm:text-sm sm:leading-5"
                 @input="refreshPasswordValidation"
               />
+              <div
+                class="hover:cursor-pointer absolute right-3"
+                @click="
+                  () => {
+                    state.showPassword.confirm = !state.showPassword.confirm;
+                  }
+                "
+              >
+                <heroicons-outline:eye
+                  v-show="state.showPassword.confirm"
+                  class="w-4 h-4"
+                />
+                <heroicons-outline:eye-slash
+                  v-show="!state.showPassword.confirm"
+                  class="w-4 h-4"
+                />
```

Comment:
use `v-if` and `v-else` instead of `v-show`?

Reply:
I think `v-show` may be better, because it may be often toggled.
![CleanShot 2022-08-26 at 18 14 35@2x](https://user-images.githubusercontent.com/56376387/186882486-25b452c7-c899-4f15-9781-9fae8e8dd020.png)

Reply:
The password toggling isn't often I think.

Reply:
PTAL